executor.go

```diff
@@ -720,7 +733,7 @@ func localExecute(pipeline *zjson.Pipeline, executionId int64, executionUuid str
 
 			jsonData, err := json.Marshal(content)
 			if err != nil {
-				fmt.Sprintf("Failed to log: %s", message)
+				log.Printf("Failed to log: %s", message)
```

Comment:
This is inside of the function that writes to the logger. Using `log.Printf` inside of this function can cause a segfault or an infinite loop

Reply:
Sprintf doesn't do anything here. I'll change it to Printf